hnn_core/tests/test_gui.py

```diff
@@ -421,6 +421,108 @@ def test_gui_adaptive_spectrogram():
     assert len(gui.viz_manager.figs[1].axes) == 2
     plt.close('all')
 
+def test_gui_layer2_dipole():
```

Comment:
Thanks for the contribution! We use flake8 for style enforcement. It failed with the following warnings: 

```
hnn_core/tests/test_gui.py:424:1: E302 expected 2 blank lines, found 1
hnn_core/tests/test_gui.py:445:1: E302 expected 2 blank lines, found 1
hnn_core/tests/test_gui.py:466:1: E302 expected 2 blank lines, found 1
hnn_core/tests/test_gui.py:487:1: E302 expected 2 blank lines, found 1
hnn_core/tests/test_gui.py:492:1: W293 blank line contains whitespace
hnn_core/tests/test_gui.py:498:33: E127 continuation line over-indented for visual indent
hnn_core/tests/test_gui.py:500:33: E127 continuation line over-indented for visual indent
hnn_core/tests/test_gui.py:507:1: E302 expected 2 blank lines, found 1
hnn_core/tests/test_gui.py:512:1: W293 blank line contains whitespace
hnn_core/tests/test_gui.py:518:33: E127 continuation line over-indented for visual indent
hnn_core/tests/test_gui.py:520:33: E127 continuation line over-indented for visual indent
hnn_core/tests/test_gui.py:525:58: W291 trailing whitespace
hnn_core/tests/test_gui.py:527:1: E302 expected 2 blank lines, found 1
``` 
Basically it wants two spaces between functions, found some indentation inconsistency, and white spaces.

You can run [flake8](https://flake8.pycqa.org/en/latest/index.html) locally in the terminal/cmd prompt if it's installed in your environment to test before you push. If you've cloned and installed the dev version of hen-core you should already have flake8 in your environment. Details about that are in our [Contributions Guide](https://jonescompneurolab.github.io/hnn-core/dev/contributing.html#installing-editable-hnn-core). Otherwise you can just pip install it. 

Let us know if you have any questions!

Reply:
Ah, I see you've already got to them! I'll deploy the checks again.

Reply:
Thank you for the suggestion. 😋 Yes, I did notice the linting issue, but I forgot to commit and push the changes earlier.